packages/dd-trace/src/plugins/outbound.js

```diff
@@ -5,18 +5,6 @@ const TracingPlugin = require('./tracing')
 
 // TODO: Exit span on finish when AsyncResource instances are removed.
 class OutboundPlugin extends TracingPlugin {
-  constructor (...args) {
-    super(...args)
-
-    this.addTraceSub('connect', message => {
-      this.connect(message)
-    })
-  }
-
-  connect (url) {
```

Comment:
Why not keep it there? If there are no publishers, this doesn't impact anything.

Reply:
I wasn't sure of the impact of subscribing to a channel, so I removed it by default. I'll revert this so we have an easy way to grab connect events in every client.